training-service/src/app/content/summary/summary.component.html

```diff
@@ -1 +1,18 @@
-<p>TWOJE ZAMÓWIONE KURSY :</p>
+<div class="container">
+    <h1>Podsumowanie</h1>
+    <p>Dziękujęmy za skorzystanie z naszych usług. Twoje zamówione kursy:</p>
+    <div class="form-check">
+    <input class="form-check-input" type="checkbox" value="" id="flexCheckChecked"checked>
+    <label class="form-check-label" for="flexCheckChecked">
+     Kurs 1
+    </label>
+    </div>
+    <div class="form-check">
+    <input class="form-check-input" type="checkbox" value="" id="flexCheckChecked" checked>
```

Comment:
value and checked are not needed attrs here

Reply:
done